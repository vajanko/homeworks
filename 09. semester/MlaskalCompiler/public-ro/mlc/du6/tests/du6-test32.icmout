ERROR: Jump out of code on IP=20
