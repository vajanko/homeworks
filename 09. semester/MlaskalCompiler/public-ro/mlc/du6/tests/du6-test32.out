Warning (line 3): Label '1' declared but never used
Warning (line 3): Label '2' declared but never used
Warning (line 3): Label '3' declared but never used
Warning (line 3): Label '4' declared but never used
Warning (line 3): Label '5' declared but never used
Error (line 3): Missing label '6' definition
There were some errors 1 and warnings 5
